### ISSUE TITLE
CID never created if Zero stops early after first init

### ISSUE THREAD

**Issue by danielmai** [dgraph-io/dgraph#3887]
If you suspect this could be a bug, follow the template.

### What version of Dgraph are you using?
v1.1.0-rc3

### Have you tried reproducing the issue with the latest release?
Reproducible on v1.0.16

### What is the hardware spec (RAM, OS)?
N/A (Linux)

### Steps to reproduce the issue (command/config used to run Dgraph).

1. Run a Dgraph Zero, and then stop it before it proposes a CID. This creates a zw directory. Stopping Zero within 1s of starting is sufficient.
2. Re-start the Dgraph Zero. Setting the log level to `-v=2` shows the anonymous telemetry logs for this issue.

```sh
# Stop after 1 second
timeout 1s dgraph zero --logtostderr -v=2
# Restart
dgraph zero --logtostderr -v=2
```

See this in the log after about 30 seconds of running, at which point it will continue logging indefinitely every minute:

```
I0830 09:58:30.781410   11848 telemetry.go:50] No CID found yet
```
**Note**: In the latest version of Dgraph (v20.03.1), this log now happens every [10 minutes](https://github.com/dgraph-io/dgraph/blob/02c90d2384f4696cf48d40a9df4319b92aff8a57/dgraph/cmd/zero/zero.go#L101-L113).

### Expected behavior and actual result.

The CID should eventually get set by Zero even if the process happened to restart early.

The CID would appear in Zero's /state:
```
{
  "counter": "4",
  "zeros": {
    "1": {
      "id": "1",
      "addr": "localhost:5080",
      "leader": true
    }
  },
  "cid": "ebd75432-db0e-4905-b798-54f63832d102",
  "license": {
    "maxNodes": "18446744073709551615",
    "expiryTs": "1592010946",
    "enabled": true
  }
}
```

When the CID isn't initialized, it is missing from /state:
```
{
  "zeros": {
    "1": {
      "id": "1",
      "addr": "localhost:5080",
      "leader": true
    }
  }
}

```

**Comment by campoy** [comment]
Hey @danielmai, what's a CID?

**Comment by danielmai** [comment]
> what's a CID?

Cluster ID. It's a UUID generated for a cluster.

**Comment by neerajbattan** [comment]
Hey @danielmai, I'm not able to reproduce it on the latest version.

**Comment by manishrjain** [comment]
Ensure that the logic is such that it would set the CID if not previously set.

**Comment by lgalatin** [comment]
Fixed in #3887 closing.
